src/controllers/users_controller.js

```diff
@@ -69,4 +69,48 @@ UsersController.post('/', async (request, response) => {
   });
 });
 
+/**
+ * Edit a user
+ */
+UsersController.put('/:userId', async (request, response) => {
+  const user = await Users.getUserById(request.params.userId);
+
+  if (!user) {
+    response.sendStatus(500);
```

Comment:
To me this feels more like a 404 user not found than an internal server error.

Reply:
Fixed.